geoengine/workflow_builder/operators.py

```diff
@@ -31,6 +31,16 @@ def to_workflow_dict(self) -> Dict[str, Any]:
             'operator': self.to_dict(),
         }
 
+    @classmethod
+    def from_workflow_dict(cls, workflow) -> 'Operator':
+        '''Returns an operator from a workflow dictionary.'''
+        if workflow['type'] == 'Raster':
+            return RasterOperator.from_operator_dict(workflow['operator'])
+        if workflow['type'] == 'Vector':
+            return VectorOperator.from_operator_dict(workflow['operator'])
+
+        raise NotImplementedError(f"Unknown workflow type {workflow['type']}")
```

Comment:
We don't have plot types in here, right?

Reply:
yes